.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+## What does this PR do?
+
+[comment]: <> (A brief explanation synthesizing the feature, bug or fix.)
```

Comment:
Also, what is the `[comment]: <>` thingy called? Seems like a cool thing to have in markdown files.